vladimir-trifonov/homework1/cv/cv.css

```diff
@@ -0,0 +1,86 @@
+
+	body{
+		background-image: url(3.jpg);
+		background-repeat: no-repeat;
+		background-attachment: fixed;
+	}
+		section
```

Comment:
@VladimirTr14 
Remember to format you code with the correct identation. It will make it easier for yourself and others to read it.